src/main/java/com/minecolonies/coremod/commands/generalcommands/RandomTeleportCommand.java

```diff
@@ -162,7 +162,7 @@ private static void teleportPlayer(final ICommandSender sender, final EntityPlay
                 }
                 else
                 {
-                    sender.getCommandSenderEntity().sendMessage(new TextComponentString("Please wait at least " + Configurations.teleportBuffer + " seconds to teleport again"));
+                    sender.getCommandSenderEntity().sendMessage(new TextComponentString("Please wait at least " + Configurations.Gameplay.teleportBuffer + " seconds to teleport again"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 186 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)